l10n_it_sdi_channel/views/company_view.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
```

Comment:
Please use `<odoo>`.
 `<data>` can be removed

Reply:
@eLBati si può mettere il tag work in progress ?

Reply:
Messo